manim/mobject/coordinate_systems.py

```diff
@@ -367,6 +362,21 @@ def construct(self):
 
         return line_graph
 
+    def origin_shift(self, axis_range: List[float]):
+        """Shifts graph mobjects to compensate when 0 is not on the axis.
+
+        Parameters
+        ----------
+        axis_range
+            The range of the axis : ``(x_min, x_max, x_step)``.
+        """
```

Comment:
Add a return value and convert to a static method
```suggestion

    @staticmethod
    def origin_shift(axis_range: List[float]) -> float:
        """Determines how to shift graph mobjects to compensate when 0 is not on the axis.

        Parameters
        ----------
        axis_range
            The range of the axis : ``(x_min, x_max, x_step)``.
```